File: Npc/11000077.cs
using Maple2.Model.Enum;
using Maple2.Script.Npc;

/// <summary>
/// 11000077: Taru
/// </summary>
public class _11000077 : NpcScript {
    internal _11000077(INpcScriptContext context) : base(context) {
        Id = 30;
    }

    public override bool Next(int selection = 0) {
        switch (Id) {
            case 0:
                // $script:0831180407000362$ 
                // - What seems to be the problem?
                return true;
            case 30:
                // $script:0831180407000365$ 
                // - How are your journeys treating you? Deployments can be exciting, but I get homesick if I'm away for too long.
                return true;
            default:
                return true;
        }
    }
}

File: Npc/11000077.py
""" 11000077: Taru """
from npc_api import Option, Script
import random


class Main(Script):
    def first(self) -> int:
        return 30

    def select(self) -> int:
        return 0

    def __0(self, pick: int) -> int:
        # $script:0831180407000362$
        # - What seems to be the problem?
        return None # TODO

    def __30(self, pick: int) -> int:
        # $script:0831180407000365$
        # - How are your journeys treating you? Deployments can be exciting, but I get homesick if I'm away for too long.
        return -1

    def button(self) -> Option:
        if (self.state, self.index) == (30, 0):
            return Option.CLOSE
        return Option.NONE
